L1Trigger/CSCTriggerPrimitives/src/CSCTriggerPrimitivesBuilder.h

```diff
@@ -81,14 +97,27 @@ class CSCTriggerPrimitivesBuilder
   /** SLHC: special switch for disabling ME42 */
   bool disableME42;
 
+  /** SLHC: special switch for the upgrade ME1/1 TMB */
+  bool runME11ILT_;
+
+  /** SLHC: special switch for the upgrade ME2/1 TMB */
+  bool runME21ILT_;
+
+  /** SLHC: special switch for the upgrade ME3/1 and ME4/1 TMB */
+  bool runME3141ILT_;
+
   int m_minBX, m_maxBX; // min and max BX to sort.
 
   /** Pointers to TMB processors for all possible chambers. */
   CSCMotherboard*
     tmb_[MAX_ENDCAPS][MAX_STATIONS][MAX_SECTORS][MAX_SUBSECTORS][MAX_CHAMBERS];
```

Comment:
Actually, just checked and `sizeof(CSCMotherboard*)` is 8 and `sizeof(std::unique_ptr<CSCMotherboard>)` is 8.  I assumed unique_ptr would have more internal members but I guess it only needs a pointer.  I'll change this to a unique_ptr array.